source/blog/database-logs.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Database Logs
+excerpt: "Now stream logs from your Aptible managed databases."
+author_name: Thomas Orozco
+author_email: thomas@aptible.com
+author_id: thomas
+posted: 2016-11-30
+section: Blog
+posts: true
+---
+We’re proud to announce that as of this week, you can now route database logs to a Log Drain, just like you’d do with app logs! This option is available when you create a new Log Drain; you can opt to send either app or database logs, or both:
+
+<p class="text-center">
+  <img class="img-responsive" src="/images/blog/database-logs/create_log_drain.png">
+</p>
+
+If you already have a Log Drain set up for apps (you should!), you can opt to either recreate it to capture both app and database logs, or simply create a new one that only captures database logs.
+
+## Why Capture Database Logs?
+
+Aptible customers have asked for database logs for  two main use cases: compliance and operations.
+
+From a compliance perspective, you can use database logs to facilitate audit logging, for example by logging sensitive queries made to your database (or all queries for that matter, if that’s a realistic option for you).
+
+From an operations standpoint, you can use them to identify new performance problems (e.g. by logging slow queries made to your database), or to better understand problems you’ve already identified (e.g. by correlating database log entries with issues you’ve experienced).
+
+## What Does My Database Log?
+
+Your database may not log what you care about out of the box. For example, Postgres is pretty quiet by default. You can usually modify logging parameters by connecting to your database and issuing re-configuration statements.
+
+For example, to enable slow query logging in Postgres >= 9.4, you’d create a database tunnel and run the following commands:
+
+**`ALTER SYSTEM SET log_min_duration_statement = 200;`**
+
+**`ALTER SYSTEM SET log_min_messages = 'INFO';`**
+
+**`SELECT pg_reload_conf();`**
```

Comment:
You might want to replace these with fences; I think we have some style that'll kick in for these. I.e. use:

    ```
    ALTER SYSTEM SET log_min_duration_statement = 200;
    ALTER SYSTEM SET log_min_messages = 'INFO';
    SELECT pg_reload_conf();
    ```